### PR TITLE
Added contributing and security guidelines

### PR DESCRIPTION
**Pull request by artem-smotrakov**
Here is a list of updates:

- Added contributing guidelines
- Added guidelines for reporting vulnerabilities
- Added SAP's Corporate Contributor License Agreement
- Updated the `README.md` with links to the new guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,17 +125,24 @@ Using local NVD cache as last update was within two hours
...
 ## Known issues
 
-Please see GitHub issues.
+Please see [GitHub issues](https://github.com/SAP/fosstars-rating-core/issues).
 
 ## How to obtain support
 
-Please create a new GitHub issue if you've found a bug, or you'd like to propose an enhancement.
+Please create a new [GitHub issue](https://github.com/SAP/fosstars-rating-core/issues)
+if you found a bug, or you'd like to propose an enhancement.
```
Comment (sourabhsparkala) [inline README.md]: Please create a new Github issue if you find a bug or if you'd like to propose an enhancement

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    * Send it by traditional letter to: OSPO Core, Dietmar-Hopp-Allee 16, 69190 Walldorf, Germany
+
+The form contains a list of employees who are authorized to contribute on behalf of your company. When this list changes, please let us know.
+
+# Commit the changes
+
+Choose the granularity of your commits consciously and squash commits that represent multiple edits or corrections of the same logical change. See the [Commit Guidelines section of Pro Git](https://git-scm.com/book/en/Distributed-Git-Contributing-to-a-Project#Commit-Guidelines) for best practices around commit messages, and use `git log` to see some examples.
+
+*  Keep the subject line to 50 characters or less if possible.
+*  Please reference the GitHub issue number in the description of the commit.
```
Comment (sourabhsparkala) [inline CONTRIBUTING.md]: Please refer the Github issue number

**Comment by artem-smotrakov** [comment]
Thanks for the review @sourabhsparkala !

**Comment by artem-smotrakov** [comment]
Thanks for the review @mibo !

**Pull request merged**